src/advanced/network.md

```diff
@@ -32,12 +32,18 @@ Full nodes are further subdivided into 2 categories:
     _light clients_.
 
 **Light clients** represent clients in the client-server paradigm; they connect
-to full nodes to retrieve information from the blockchain they are
-interested in, and to send transactions. Exonum provides a
-[“proofs mechanism”](../glossary.md#merkle-proof),
+to full nodes for several purposes:
+
+- to [retrieve](node-management.md#explorer-api-endpoints) information they are
+  interested in from the blockchain
```

Comment:
Nit: dot missing ?

Reply:
each point here is a continuation of the sentence "**Light clients** represent clients in the client-server paradigm; they connect to full nodes for several purposes:"
The dot should be put only in the end of the sentence, i.e. after the last point only.